packages/puppeteer-core/src/api/Page.ts

```diff
@@ -2672,8 +2672,10 @@ export abstract class Page extends EventEmitter<PageEvents> {
    * ```
    *
    * Shortcut for {@link Frame.click | page.mainFrame().click(selector[, options]) }.
-   * @param selector - A `selector` to search for element to click. If there are
-   * multiple elements satisfying the `selector`, the first will be clicked
+   * @param selector - A
+   * {@link https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors | selector}
+   * of an element to type into. If there are multiple elements satisfying the
```

Comment:
I think this should say "to click" instead of "to type into"?

Reply:
Shoot sorry thats a dumb typo